src/components/panel/panel.spec.js

```diff
@@ -344,6 +344,30 @@ describe('$mdPanel', function() {
       // expect(panelRef).toBeUndefined();
       expect(PANEL_EL).not.toExist();
     }));
+
+    it('should focus on open', function() {
+      var htmlContent = 'Donuts';
```

Comment:
This htmlContent doesn't matter for this test. You can just add it directly to the template. Same with the below test.